src/lib/sys.c

```diff
@@ -244,6 +255,13 @@ bool file_is_executable(const char *filename)
 	return false;
 }
 
+bool sys_file_is_hardlink(const char *file1, const char *file2)
+{
+	struct stat sb, sb2;
+
+	return !(lstat(file1, &sb) == 0 && lstat(file2, &sb2) == 0 && sb.st_ino == sb2.st_ino);
```

Comment:
why are you negating this? if both inodes are the same doesn't that mean it is a hardlink?

Reply:
Wow, this is wrong. Not is wrong here and where I call it, so it's working, but I'll fix that.